slp-postage-protocol.md

```diff
@@ -0,0 +1,268 @@
+![Simple Ledger Protocol](images/SLP-logo-solid-200.png)
+
+# Simple Ledger Postage Protocol
```

Comment:
This is so cool @vinarmani , thank you for working on this.  You clearly spent a lot of time working on it already.  Great job on the spec so far.  I don't see any major issues, my main viewpoint is coming from an optimizing UX around allowing a wallet to shop between multiple post-offices.